docs/http/jwt-validation.mdx

```diff
@@ -0,0 +1,106 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# JWT Validation
+
+## Overview
+
+The JWT Validation module allows or denies traffic based on validation of the JSON Web Token (JWT) provided within the request that
+was initiated to your ngrok endpoints. You will define the claims used to validate the token in conjunction with verifying it has been properly signed by the issuer.
+
+A connection is allowed only if it has been signed by the issuer and the defined claims match.
+
+:::note Coming Soon
+
+This documentation is incomplete. Please check back later, we appreciate your
+patience.
+
+**You can consult the [JWT Validation API documentation](https://ngrok.com/docs/api/resources/edge-route-jwt-validation-module/) in the interim.**
+:::
+
+<Tabs groupId="connectivity" queryString="cty">
+	<TabItem value="agent-cli" label="Agent CLI" default>
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="agent-config" label="Agent Config">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="ssh" label="SSH">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="go-sdk" label="Go">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="javascript-sdk" label="Javascript">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="python-sdk" label="Python">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="rust-sdk" label="Rust">
+		Currently Unavailable
+	</TabItem>
+	<TabItem value="k8s" label="Kubernetes Controller">
+		Currently Unavailable
+	</TabItem>
+</Tabs>
+
+## Behavior
+
+### Multiple Issuers
+
+You may specify multiple Issuers to be used for JWT validation. A request
+is considered validated if it presents a JWT signed by any of the
+specified issuers.
+
+### Multiple Audience Claims
+
+You may specify multiple audience claims to be used for JWT validation. JWT validation will require one of the audience claims to be present within the JWT.
+
+### Multiple JWS
+
+You have the ability to provide multiple JWKS urls. During JWT validation the list of JWKS provided will be used to validate the JWT.
```

Comment:
do we want to talk about the allowed algorithms as well?

Reply:
added mention